### DISCUSSION TITLE
Copilot does not activate on fresh install of VSCode (Mac M1).

### DISCUSSION THREAD

**Discussion by rehypothecation** [community/community#41955] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

As title says, Copilot does not activate on a fresh install of VSCode.

Steps to reproduce:

1. Download VSCode on Mac (Apple Silicon version).
2. Navigate to any file that Copilot supports.
3. Nothing happens.

<img width="1024" alt="Screenshot 2022-12-16 at 11 58 25 PM" src="https://user-images.githubusercontent.com/72181637/208225938-125621b2-43ee-40ac-af43-328cd94eabef.png">

**Comment by motravo** [comment, marked as answer]
Same here, on an M1. Just started my free trial and ... it doesn't work.
Any workaround?

**Reply by motravo** [reply]
Workaround: pull down the Uninstall menu and choose Install Another Version, and install 1.63.7601. Then, the authentication popup appears in the lower-right corner of the window and you can login.

**Comment by iamcjwilliams** [comment]
@rehypothecation Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code
